p4cmd/p4cmd.py

```diff
@@ -81,45 +82,46 @@ def run_cmd2(self, cmd, args=[], use_global_options=True, online_check=True):
         if online_check:
             if not self.host_online():
                 logging.warning("Can't connect to %s on port %s" % (self.__server_address(), self.__port_number()))
-                #raise p4errors.ServerOffline("Can't connect to %s on port %s" % (self.__server_address(), self.__port_number()))
 
         if self.perforce_root is not None:
             os.chdir(self.perforce_root)
 
-        # build arg strings within the max size
+        file_list = [f'"{f}"' for f in file_list]
+
+        # build arg and file strings within the max size
         clamped_arg_list = split_list_into_strings_of_length(args, max_length=MAX_ARG_LEN)
```

Comment:
ah we probably don't need to clamp the arg list right? just using args straight up means we can skip one level of indentation